README.md

```diff
@@ -30,6 +30,36 @@ python ./evaluate.py -m mace_mobilenetv1_nchw -t float32 -k android --target="op
 ```
 Refer to the below instructions for running the `scripts/evaluate.py` script for more information
 
+## Run models with dynamic shape through VM
+In the table below you can see the performance numbers for a set of models from Onnx model zoo which were achieved on the `Qualcomm Snapdragon 8 Gen 1`:
+
+|                  |   onnx_ssd_resnet34   |   onnx_yolo_v3   | onnx_faster_rcnn |   mace_mobilenetv1_nchw  | mace_resnet50_v2 |
+|------------------|-----------------------|------------------|------------------|--------------------------|------------------|
+| TVM FP16    (GE) |                 160,69|             47,38|            203,98|                      3,04|             24,53|
+| TVM FP16    (VM) |                 496,83|             66,44|            208,31|                      3,11|             24,63|
```

Comment:
VM is 3x slower than GE on some of the runs. Is that just from the NMS degradation? Seems like a large perf gap to be caused just by the NMS layer.

Reply:
@sabez30 caught me up on the issue, no worries